terrain/instancing/voxel_instance_library_item.cpp

```diff
@@ -263,10 +279,17 @@ void VoxelInstanceLibraryItem::_bind_methods() {
 	ADD_PROPERTY(PropertyInfo(Variant::OBJECT, "material_override", PROPERTY_HINT_RESOURCE_TYPE, "Material"),
 			"set_material_override", "get_material_override");
 
+	ADD_PROPERTY(PropertyInfo(Variant::INT, "cast_shadow", PROPERTY_HINT_ENUM, "Off,On,Double-Sided,Shadows Only"), "set_cast_shadows_setting", "get_cast_shadows_setting");
+
 	ADD_PROPERTY(PropertyInfo(Variant::INT, "collision_layer", PROPERTY_HINT_LAYERS_3D_PHYSICS),
 			"set_collision_layer", "get_collision_layer");
 	ADD_PROPERTY(PropertyInfo(Variant::INT, "collision_mask", PROPERTY_HINT_LAYERS_3D_PHYSICS),
 			"set_collision_mask", "get_collision_mask");
 
 	BIND_CONSTANT(MAX_MESH_LODS);
+
+	BIND_ENUM_CONSTANT(VisualServer::SHADOW_CASTING_SETTING_OFF);
+	BIND_ENUM_CONSTANT(VisualServer::SHADOW_CASTING_SETTING_ON);
+	BIND_ENUM_CONSTANT(VisualServer::SHADOW_CASTING_SETTING_DOUBLE_SIDED);
+	BIND_ENUM_CONSTANT(VisualServer::SHADOW_CASTING_SETTING_SHADOWS_ONLY);
```

Comment:
Does this binds the constants properly in the doc and completion?

Reply:
Binding them ended up being unnecessary.  I added the entry to the docs and completion works in gdscript.